input/groups/emote.json

```diff
@@ -1,29 +1,36 @@
 {
   "topLevelCategories": {
+    "Core": {
+      "Standalone: Other": [
+        "Bless"
```

Comment:
So... thoughts on how to handle Wizard Vault unlocks in general. For some other wizard vault skins, I've currently got them under SotO, but they're not technically expansion specific. But they're also part of the current Wizard Vault cycle, which will end for the next expansion. I guess I'm leaning on placing them in SotO at the moment and reassessing once we see how they are handled on rotation.

Reply:
I first gave them their own category, right above Gem Store, but then I noticed many of the items require Secrets of the Obscure (wiki has an updated list), so I just moved them under SotO as well.

My guess is most exclusive Wizard's Vault rewards will remain expansion-specific, even when they go out of rotation. If they don't, then creating a separate category might become interesting once again.